Modules/System/Assisted Setup/src/AssistedSetupImpl.Codeunit.al

```diff
@@ -99,6 +99,18 @@ codeunit 1813 "Assisted Setup Impl."
         AssistedSetup.Modify(true);
     end;
 
+    procedure Reset(ExtensionID: Guid; PageID: Integer)
```

Comment:
What is the ExtensionID used for here? I fail to find the spot where that parameter used. Doesn't seem to be part of the Primary Key on the Assisted Setup table. Maybe we can omit it?

Reply:
Just spoke to Dutta. I understand that we had to keep this parameter even though it is not needed, as removing it would be a breaking change. I've created an internal bug to overload the methods without the unused parameter and to deprecate the ones which have the ExtensionID. In this case, I would get rid of the parameter right away. There is no need to introduce a new function which we deprecate right away.

Reply:
Do you then introduce a function with Reset(PageID:Integer) or will you even omit that function?